### PR TITLE
Lots of changes

### PR DESCRIPTION
**Pull request by farhaven**
I can split these out into separate MRs if necessary, but they should all be noncontroversial.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 ---
-
 letsencrypt_install_dir: /opt/letsencrypt/
 letsencrypt_cert_dir: /etc/ssl/letsencrypt/
 letsencrypt_webroot_dir: /var/www/acme-challenges/
 letsencrypt_skip_selftest: False
 letsencrypt_email: contact@example.com
-letsencrypt_cert_domains:
-  example.com: [www.example.com, foo.example.com]
```
Comment (RonjaPonja) [inline defaults/main.yml]: I'm a friend of have all relevant variables one might want to overwrite in the defaults/main.yml. I see why you wouldn't want to have this enabled though. Would it be acceptable to keep, but comment out letsencrypt_email and letsencrypt_cert_domains?

File: tasks/main.yml [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
 - name: Add user letsencrypt
   user: name=letsencrypt system=yes group=ssl-cert state=present home={{ letsencrypt_install_dir }} shell=/bin/bash
 
-- name: Let letsencrypt user reload nginx
-  lineinfile: dest=/etc/sudoers line="letsencrypt ALL = NOPASSWD{{':'}} /bin/systemctl reload nginx"
+- name: "Let letsencrypt user reload {{ letsencrypt_web_server }} (systemctl) "
+  lineinfile:
+    dest: /etc/sudoers
+    line: "letsencrypt ALL = NOPASSWD{{':'}} /bin/systemctl reload {{ letsencrypt_web_server | default('nginx') }}"
+  when: ansible_service_mgr == "systemd"
```
Comment (RonjaPonja) [inline tasks/main.yml]: Can we define this as the default in defaults/main.yml?

File: tasks/main.yml [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
+
+- name: "Let letsencrypt user reload {{ letsencrypt_web_server }} (SysV init)"
+  lineinfile:
+    dest: /etc/sudoers
+    line: "letsencrypt ALL = NOPASSWD{{':'}} /etc/init.d/{{ letsencrypt_web_server | default('nginx') }} reload"
+  when: ansible_service_mgr != "systemd"
```
Comment (RonjaPonja) [inline tasks/main.yml:22]: Hmmm. Maybe we could just always add both lines?.. but I'm not sure.

**Review by RonjaPonja — CHANGES_REQUESTED** [review]
Looks fine. I'd like to at least address the defautl/main.yml changes though.

**Comment by HerrSpace** [comment]
Opinions? I will make the relevant changes if you think they are okay.

**Comment by RonjaPonja** [comment]
Ahh, a well aged merge request. Good time to merge this, right? RIGHT?

**Pull request merged**